<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDFTuner</class>
 <widget class="QWidget" name="PDFTuner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>141</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Тип распределения:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="boxDistrs"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Метод:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="boxMethods"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Толщина линии</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Цвет линии</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ColorLabel" name="lblPDFColor">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="DoubleNumberEdit" name="sbxLineWidth">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleNumberEdit</class>
   <extends>QLineEdit</extends>
   <header>extra_widgets/numberedit.h</header>
  </customwidget>
  <customwidget>
   <class>ColorLabel</class>
   <extends>QLabel</extends>
   <header>extra_widgets/colorlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
